lib/create-app.js

```diff
@@ -1,9 +1,13 @@
-const copy = require('kopy');
 const path = require('path');
+const fs = require('fs-extra');
 const kebabCase = require('just-kebab-case');
 const semver = require('semver');
 const pascalCase = require('just-pascal-case');
-const { version: currentCLIVersion } = require('../package.json');
+const simpleGit = require('simple-git/promise');
+const { version: currentCLIVersion, uiTemplateRepository } = require('../package.json');
+
+const COOKIECUTTER_PREFIX = '__';
+let ARGS = {};
```

Comment:
Any preference for this global variable instead of passing `ARGS` into the function that consumes it?  Looks like its only used by `replaceTemplateValues`.  It would make the function more testable too!

Reply:
I agree that local variable would be preferred. I'll switch to that.